features/dd-sdk-android-rum-debug-widget/src/main/kotlin/com/datadog/android/insights/LocalInsightOverlay.kt

```diff
@@ -3,6 +3,7 @@
  * This product includes software developed at Datadog (https://www.datadoghq.com/).
  * Copyright 2016-Present Datadog, Inc.
  */
+
```

Comment:
I don't see any unit tests for the new module. Are we going to add them later on?